### ISSUE TITLE
Can not download last version of source code from cargo with "validate_with" function.

### ISSUE THREAD

**Issue by ynhhoJ** [jprochazk/garde#113]
I run into a problem where installing `garde` dependency from crates.io / `cargo` does not have last version.

Here is an example:

`cargo add garde -F full`

`Cargo.toml`
```toml
garde = { version = "0.18.0", features = ["full"] }
````

`validate_with` - does not exist in `validate.rs`
<details>
  <summary>validate.rs</summary>
  
![image](https://github.com/jprochazk/garde/assets/22500212/1ab5bcdd-4ac7-4ea3-a26a-63a85c5d760d)
</details>

<hr />

Installing directly from GitHub fixes problem:
`Cargo.toml`
```toml
garde = { version = "0.18.0", features = [
  "full",
], git = "https://github.com/jprochazk/garde.git", branch = "main" }
```

`validate_with` - exist in `validate.rs`

<details>
  <summary>validate.rs</summary>
  
![image](https://github.com/jprochazk/garde/assets/22500212/37bc1fb3-8762-48c7-ba47-97ce5a7d9fff)
</details>

**Comment by ynhhoJ** [comment]
Also i tried to remove `grade` folder from `.cargo/registry/src...` but it does not helped me.

**Comment by jprochazk** [comment]
In general, if depending on the `main` git branch is possible for you, then it's fine to do just that.

The `validate_with` feature was added a little while ago, but was not released until [v0.19.0](https://github.com/jprochazk/garde/releases/tag/v0.19.0), which I published today.
